edison/templates/policy.html

```diff
@@ -0,0 +1,139 @@
+                <option>children room 1</option>
+                <option>children room 2</option>
+              </select>
+            </div>
+            <div class="form-row">
+              <div class="form-group col-md-4">
+                <label for="light">Light</label>
+                <select class="form-control" id="light">
+                  <option> </option>
+                  <option>
+                    On </option>
+                  <option>Off</option>
+                </select>
+              </div>
+              <div class="form-group col-md-4">
+                <label for="ac">AC</label>
+                <select class="form-control" id="ac">
+                  <option> </option>
+                  <option>
+                    On </option>
+                  <option>Off</option>
+                </select>
+              </div>
+              <div class="form-group col-md-4">
+                <label for="shutters">Shutters</label>
+                <select class="form-control" id='shutters'>
+                  <option> </option>
+                  <option>
+                    On </option>
+                  <option>Off</option>
+                </select>
+              </div>
+            </div>
+            <div class="form-row">
+              <div class="form-group col-md-4">
+                <button class="btn btn-link collapsed" id='addCondition' data-toggle="collapse"
+                  data-target="#collapseAddCondition" type="button" aria-expanded="false"
+                  aria-controls="collapseAddCondition">
+                  Add Condition
+                </button>
+              </div>
+            </div>
+            <div class="collapse" id="collapseAddCondition" style="width: 50rem;">
+              <div class="form-row">
+                <div class="form-group col-md-5">
+                  <label for="sensor">Sensor</label>
+                  <select class="form-control" id="sensor">
+                    <option id='sensorOption0'> </option>
+                    <option id='sensorOption1'>
+                      Humidity </option>
+                    <option id='sensorOption2'>Light</option>
+                    <option id='sensorOption3'>Tempetrue</option>
+                  </select>
+                </div>
+                <div class="form-group col-md-5">
+                  <label for="condition">Condition</label>
+                  <select class="form-control" id="condition">
+                    <option> </option>
+                    <option>
+                      < </option>
+                    <option>=</option>
+                    <option>></option>
+                    <option>Between</option>
+                  </select>
+                </div>
+              </div>
+              <div id='conditionTerm'>
+              </div>
+              <div class="form-row">
+                <div class="form-group col-md-4" id="showValue" style="display:none;">
```

Comment:
you want that each time I change the class of the object? instead changing the display value? 
so that will be duplicate classes ?

Reply:
Sorry didn't understand what you mean.
Instead of writing `style="display:none;"` inside the div tag, create the following class:
```
.hide {
  display: none;
}
```

And then add this class to the list of classes this div uses:

```
<div class="hide form-group col-md-4" id="showValue">
```

If you wish to display this div, create the following class:

```
.show {
  display: block;
}
```

And then use JavaScript to display it:

```
var showValueDiv = document.getElementById('showValue');

if(showValueDiv.classList.contains('hide')) {
  showValueDiv.classList.add('show');
  showValueDiv.classList.remove('hide');
} else if (showValueDiv.classList.contains('show')) {
  showValueDiv.classList.add('hide');
  showValueDiv.classList.remove('show');
}
```

Reply:
this is contradict to the  class option (the PolicyElementsModifier) , maybe I used a class name that is common to all the options and I change this class according to the element ID and the class ?

Reply:
I don't see a connection here to the PolicyElementsModifier.
This class solved the issue of having many configuration options and organize them.
In this situation you have only two options hide and show, so its okay to use if else.

Reply:
this elements are the same elements that we configure in the PolicyElementsModifier, If I do this I still using a multiple If and else statements  and the  PolicyElementsModifier is not relevant

Reply:
Oh yea sorry, this PR is so big that its hard to follow.
The main thing that I said was not to put CSS hard coded inside html files.
The second thing was a suggestion for future code in case you would need to play elements visibility. So you don't have to use it here.

Reply:
OK, thanks!!